tests/src/test/scala/whisk/core/entity/test/SchemaTests.scala

```diff
@@ -70,6 +69,46 @@ class SchemaTests extends FlatSpec with BeforeAndAfter with ExecHelpers with Mat
     a[DeserializationException] should be thrownBy Privilege.serdes.read("???".toJson)
   }
 
+  behavior of "TransactionId"
+
+  it should "serdes a transaction id without extraLogging parameter" in {
+    val txId = 4711
```

Comment:
the `id` is accessible via `TransactionId.id`

Reply:
Changed to use ...meta.id